Makefile

```diff
@@ -74,10 +74,10 @@ update-devel-image:
 	@ln Containerfile LICENSE LICENSE.3RD_PARTY $(TMPDIR)
 ifeq ($(shell uname),Linux)
 	@CGO_ENABLED=1 GOOS=linux go build -race -o $(TMPDIR)/forwarder ./cmd/forwarder
-	@$(CONTAINER_RUNTIME) buildx build -f Containerfile --build-arg BASE_IMAGE=ubuntu:latest -t saucelabs/forwarder:$(TAG) $(TMPDIR)
+	@$(CONTAINER_RUNTIME) buildx build --network host -f Containerfile --build-arg BASE_IMAGE=ubuntu:latest -t saucelabs/forwarder:$(TAG) $(TMPDIR)
```

Comment:
```
make update-devel-image CONTAINER_RUNTIME=docker                                                             2 ↵
unknown flag: --network
See 'docker --help'.
```
:/

Reply:
That's strange it I do not see any errors in the action

https://github.com/saucelabs/forwarder/actions/runs/8522801578/job/23343764480

Reply:
It could've been my setup. If it works with pure docker then ok.